cluster/common.sh

```diff
@@ -475,7 +518,7 @@ function verify-kube-binaries() {
 # If KUBERNETES_SKIP_CONFIRM is set to y, we'll automatically download binaries
 # without prompting.
 function verify-release-tars() {
-  if ! $(find-release-tars); then
+  if ! find-release-tars; then
```

Comment:
this is a behavioral change, is find-release-tars in the current shell ok? the previous version runs in a subshell which means EG the environment variables are isolated.

Reply:
I examine the project, there is no variable overlapping in this function, this change should be ok.

Reply:
what about `exit` ?

Reply:
looks like that's not an issue